osquery/tables/system/darwin/homebrew_packages.cpp

```diff
@@ -43,6 +53,9 @@ std::string getHomebrewNameFromInfoPlistPath(const std::string& path) {
   return bits[bits.size() - 1];
 }
 
+// Homebrew formulas are under a path like
+// /usr/local/Cellar/<formula>/<version>/* passing the formula path will return
```

Comment:
On my machine, this path is `/opt/homebrew/Cellar/<formula>/<version>/.brew/*` Is the addition of `.brew` an issue?

Reply:
So that would be the case for non-cask installs (under `/opt/homebrew/Cellar/`), which I did not touch at all. I focused only on `/opt/homebrew/Caskroom`